### PR TITLE
chore: fixes

### PR DESCRIPTION
**Pull request by squakez**

### CODE REVIEW

**Comment by squakez** [comment]
@asmigala mind having a look?

**Review by oscerd — APPROVED** [review]

File: readme.didact.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ In order to execute this demo, you will need to have an OpenShift cluster with t
 
 Apart from the support provided by the VS Code extension, you also need the Apache Camel K CLI ("kamel") in order to access all Camel K features.
 
-[Check if the Apache Camel K CLI ("kamel") is installed](didact://?commandId=vscode.didact.requirementCheck&text=kamel-requirements-status$$kamel%20version$$Camel%20K%20Client&completion=Apache%20Camel%20K%20CLI%20is%20available%20on%20this%20system. "Tests to see if `kamel version` returns a result"){.didact}
+[Check if the Apache Camel K CLI ("kamel") is installed](didact://?commandId=vscode.didact.cliCommandSuccessful&text=kamel-requirements-status$$kamel%20version$$Camel%20K%20Client&completion=Apache%20Camel%20K%20CLI%20is%20available%20on%20this%20system. "Tests to see if `kamel version` returns a result"){.didact}
```
Comment (asmigala) [inline readme.didact.md]: This one can stay as `requirementCheck`, it's the one above (`Check if you're connected to an OpenShift cluster`) that needs to be `cliCommandSuccessful`

Comment (squakez) [reply readme.didact.md]: Yeah, I am not very familiar with their difference, but I understood the `cliCommandSuccessful` is looking for a 0 return from the command.

**Comment by squakez** [comment]
While looking for the documentation of the `cliCommandSuccessful` I found how to create parallel terminal windows and closing them. I've enhanced the quickstart with that too. Please, @asmigala, check it out.

**Comment by asmigala** [comment]
@squakez the two terminals work well, including the stopping command which is very nice.

The issue with the check is still present, currently it looks like this:

image

The middle one need to be changed to `cliCommandSuccessful`, the remaining two are ok

**Comment by squakez** [comment]
The problem was of a different nature. The `cliCommandSuccessful` is just checking if the command return a 0 result, whilst the `requirementCheck` is looking for a text with a certain value and must be specified as part of the command (`NAME ...` is what we expect the command to printout in our example). It was a good learning excercise of didact after all :)

Please have a further look to make sure we're good to go now, thanks!

**Comment by asmigala** [comment]
lgtm, can you please also rebase on `1.4.x`? Also, I don't have merge rights, can somebody please merge this?

**Comment by heiko-braun** [comment]
@oscerd @squakez Can you backport this to 1.4.x?

**Comment by oscerd** [comment]
Yes, sure. I don't know if @squakez is around today, but I can definitely backport it

**Comment by squakez** [comment]
Just back from PTO, trying to get up to speed with the mail backlog... :) I see you already backported @oscerd, thanks! Let me know if something else is needed.

**Pull request merged**